.pre-commit-config.yaml

```diff
@@ -0,0 +1,20 @@
+repos:
+-   repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.3.0
+    hooks:
+    -   id: check-yaml
+    -   id: end-of-file-fixer
+    -   id: trailing-whitespace
+        exclude: \.svg$
+-   repo: https://github.com/pycqa/flake8
+    rev: 5.0.4
+    hooks:
+    -   id: flake8
+        exclude: seaborn/(cm\.py|external/)
+        types: [file, python]
+-   repo: https://github.com/pre-commit/mirrors-mypy
```

Comment:
Just curious but what's the rationale for pointing at this repo vs the main mypy one?

Reply:
The mirrors-mypy repo provides a [hook](https://github.com/pre-commit/mirrors-mypy/blob/main/.pre-commit-hooks.yaml) that can be used out of the box, but the mypy repo itself doesn't do so (you would have to supply all the logic to run it in this YAML file).